### PR TITLE
Added a new moth customisation set to the character setup

### PR DESCRIPTION
**Pull request by Dalii1**
## About The Pull Request
This PR adds a new moth customisation set and makes it available in the character setup menu. 

Introducing, the lightbearer moth!
image
image

Setup menu preview:

image
image
image

## Why It's Good For The Game
Adds more variety to moth customisation!

## Changelog
:cl:
add: added the lightbearer moth set, available in the character setup.
image: added icons for the lightbearer set; new moth wings, antennae and markings.
/:cl:

### CODE REVIEW

**Comment by ArcaneDefence** [comment]
image
It's covering the arm here

**Comment by SomeoneYouProbablyKnow** [comment]
CUTE!!! A cool bright moth set. Very nice

**Comment by ZephyrTFA** [comment]
did you get approval from a maintainer for this

**Comment by Jacquerel** [comment]
They now have permission from me because I don't think new sprites for species body parts that we already have is a feature you should need much or really any approval for 

(Still also needs sprite lead approval though)

**Comment by DaliIsTaken** [comment]
Thank you jacquerel. I am not very aware of the spriting approval process.

**Comment by DaliIsTaken** [comment]
> image It's covering the arm here

Hello, the pixel placement at the arm is meant to be consistent with the other wings in profile (though some of the wings do diverge from this). I'll exemplify with the atlas wings: 
image

**Review by Wallemations — APPROVED** [review]

**Comment by Wallemations** [comment]
not really sure if I'm allowed to merge this one

File: code/datums/sprite_accessories.dm
Comment (Jacquerel) [inline code/datums/sprite_accessories.dm:2413]:
```suggestion
	icon_state = "lightbearer"

```

**Review by Jacquerel — CHANGES_REQUESTED** [review]

**Pull request merged**
